emulators/mame/Portfile

```diff
@@ -125,9 +159,15 @@ depends_run-append  \
 # Build Options
 #------------------------------------------------------------------------------
 
-use_parallel_build  yes
+# Note: ARM64 support untested
+#supported_archs     ppc ppc64 i386 x86_64 arm64
+supported_archs     ppc ppc64 i386 x86_64
```

Comment:
usually, we only exclude archs that are known not to work, rather than ones we don't know whether they work or not. Principle is to be more inclusive rather than exclusive, in general.

Reply:
Makes sense. In this case, the Mame makefile doesn't yet support MacOS builds for ARM64... and builds are guaranteed to fail. Tackling this is next on my to-do list, though.

Given that, should this be changed, or left as-is?

Reply:
Since ARM64 will be tackled over the next two weeks, can we leave this as-is for now?